SQL/migrate-2023/V011__codeside_cargo.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- Creates a new cargo order table for use with new codeside cargo (#20030)
+--
+
+ALTER TABLE ss13_cargo_orderlog
+    ADD INDEX (order_id);
```

Comment:
That wont work.
The order id is not unique across multiple rounds.
It is possible (and very likely) that multiple duplicate order ids exist in the database, as the "start order id" is a randomly generated (afaik 4 digit) number at the start of a round.